src/main/java/com/musibondi/resource/PlaylistResource.java

```diff
@@ -18,57 +20,39 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PlaylistResource.class);
```

Comment:
You can use Lombok's @Slf4j on the class and it'll create the logger instead of explicitly declaring it here. Then you just call it as, e.g. `log.info(...)`

Reply:
Woohoo! Great tip, thanks! will include before merge